install.c

```diff
@@ -473,7 +544,7 @@ int inst_choose_partition(char **partition, int swap, char *txt_menu, char *txt_
     items2[item_cnt2++] = strdup(buf);
   }
 
-  if(swap) {
+  if((flags & ICP_SWAP)) {
```

Comment:
How about adding a test macro since this seems to be used multiple times? TEST_ICP_SWAP() or something like that? Similar with TEST_ICP_BLOCK_ALL()?